document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/cpu.md

---
path: "/docs/create-installer/host-preflights/cpu"
date: "2022-01-13"
weight: 25
linktitle: "CPU"
title: "CPU"
---
 

> The CPU host preflight check can be used to detect and validate the total number of CPU cores on the machine.

Reviewer: ```suggestion
The CPU host preflight check is used to detect and validate the total number of CPU cores on the machine.
```
